content/en/docs/setup/learning-environment/minikube.md

```diff
@@ -405,13 +405,13 @@ Some drivers will mount a host folder within the VM so that you can easily share
 Host folder sharing is not implemented in the KVM driver yet.
 {{< /note >}}
 
-| Driver | OS | HostFolder | VM |
-| --- | --- | --- | --- |
-| VirtualBox | Linux | /home | /hosthome |
-| VirtualBox | macOS | /Users | /Users |
-| VirtualBox | Windows | C://Users | /c/Users |
-| VMware Fusion | macOS | /Users | /mnt/hgfs/Users |
-| Xhyve | macOS | /Users | /Users |
+| Driver        | OS      | HostFolder   | VM                 |
+| --------------|---------|--------------|--------------------|
+| VirtualBox    | Linux   | ``/home``    | ``/hosthome``      |
+| VirtualBox    | macOS   | ``/Users``   | ``/Users``         |
```

Comment:
@lightlace , thanks for this PR. 
- Are two back ticks required to format the directory names in the table? Or could you use one back tick?
- Could you add a table caption shortcode for this table. The table should have a caption.
This page explains how to use the table caption shortcode.
https://deploy-preview-18748--kubernetes-io-master-staging.netlify.com/docs/contribute/style/hugo-shortcodes/#table-captions

For example, you could add something like:

```
{{< table caption = "Drivers, operating systems, mounted host folders" >}}
... the table

{{< /table */>}}

```

Reply:
I'd expect to see single backticks.